### ISSUE TITLE
[Feature request] Support for HDR colors beyond 1.0

### ISSUE THREAD

**Issue by ividyon** [PixiEditor/ColorPicker#48]
Hey,

in 32-bit color situations, or HDR color situations, RGBA values can go beyond 1.0f. This currently isn't really supported in the color picker. Would it be much work to implement?

For the color preview, the displayed color could just be clamped by the highest value.

Photoshop also has an interesting implementation for working with 32-bit colors. Clip attached below.

https://github.com/user-attachments/assets/358df881-399d-496e-a5e9-706828d95bfb

**Comment by flabbet** [comment]
I agree this would be a great feature. Although our development cycle is tightly bound to [our image editor PixiEditor](https://github.com/PixiEditor/PixiEditor). We might implement this only if we decide to add HDR features. This probably will happen, but I can't tell you when or promise if at all.

I recommend forking the repo and adding this yourself if you need it.
